Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/.template.config/template.json

```
@@ -75,5 +79,50 @@
             "path": "Company.FunctionApp.fsproj"
         }
     ],
-    "defaultName": "Company.FunctionApp"
+    "defaultName": "Company.FunctionApp",
+    "postActions":
+    [
+        {
+            "condition": "(NetCore && FrameworkShouldUseV1Dependencies)",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
+            "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+            "ContinueOnError": "true",
+            "ManualInstructions": [],
+            "args": {
+                "referenceType": "package",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "1.3.2",
```

Is this formatting intentional? Or should it be separate lines?

```suggestion
                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore",
                "version": "1.3.2",
```

Same question for the other lines in this file

---

Oh whoops we had that before but it should indeed have been changed. Will fix now.